testsupport/wait/member.go

```diff
@@ -68,6 +68,13 @@ func NewMemberAwaitility(cfg *rest.Config, cl client.Client, ns, clusterName str
 	}
 }
 
+// WaitForMetricsService verifies that there is a service called `host-operator-metrics-service`
+// in the member namespace.
+func (a *MemberAwaitility) WaitForMetricsService(t *testing.T) {
+	_, err := a.WaitForService(t, "member-operator-metrics-service")
+	require.NoError(t, err, "failed while waiting for 'member-operator-metrics-service' service")
+}
+
```

Comment:
maybe I've missed it, but I can't find where this Wait function is being invoked now.

Reply:
yeah, I need to make sure it's eventually called, but for now we don't expose (and assert) custom metrics on member-operator, AFAICS